### PR TITLE
8 build fake data in database

### PR DESCRIPTION
**Pull request by AhmedAbuSharar**

### CODE REVIEW

File: server/database/seeds/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/extensions */
```
Comment (Raghad-Mezied) [inline server/database/seeds/index.ts]: Edit the roles and delete this line

File: server/database/config/build.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+//   await Promise.all([
+//     buildModelData(adminProducts, adminProductsModel),
+//     buildModelData(productsPharmacies, productsPharmaciesModel),
+//     buildModelData(productsRequests, productsRequestsModel),
+//     buildModelData(salesHistory, salesHistoryModel),
+//   ]);
+// };
+
+// buildFakeData();
```
Comment (Raghad-Mezied) [inline server/database/config/build.ts]: Delete the commented code if you don't need them.

File: server/database/config/build.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  productsPharmacies,
+  productsRequests,
+  salesHistory,
+} from '../seeds';
+
+// console.log(sequelize);
+const buildDB = async () => {
+  // console.log('build');
+  await sequelize.sync({ force: true });
+  // console.log(pharmacies);
```
Comment (Raghad-Mezied) [inline server/database/config/build.ts]: remove the consoles

File: server/database/config/build.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  adminProducts,
+  pharmacies,
+  products,
+  productsPharmacies,
+  productsRequests,
+  salesHistory,
+} from '../seeds';
+
+// console.log(sequelize);
+const buildDB = async () => {
```
Comment (Raghad-Mezied) [inline server/database/config/build.ts]: Good Job, I need you to create two functions, one to build the tables and another to insert your fake data.
And add to scripts to run them.

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Review by Ibtisam-Hemmo — APPROVED** [review]

**Review by Nader-SH — APPROVED** [review]

**Pull request merged**